src/components/album/AlbumPreview.js

```diff
@@ -25,28 +30,65 @@ const AlbumPreview = ({
       return;
     }
     //extract the album id
-    const albumId = match[1];
+    const spotifyAlbumId = match[1];
+
     //  Spotify album URL for embedding
-    const spotifyAlbumUrl = `https://open.spotify.com/embed/album/${albumId}`;
+    const spotifyAlbumUrl = `https://open.spotify.com/embed/album/${spotifyAlbumId}`;
     setSpotifyEmbedUrl(spotifyAlbumUrl);
-  }, [spotifyUrl]);
 
+    // Fetch MongoDB album ID from  API
+    // AKOS: how would I get the id of the album in this album preview component to pass it later  to ReviewsList.js as a prop on line 68?
+    // mongo album id is not being set and instead of it it passes spotify id
+    axiosInstance
+      .get(`/albums/${albumId}`)
+      .then((response) => {
+        console.log(response.data.album._id);
+
+        setMongoAlbumId(response.data.album._id);
+      })
+      .catch((error) => {
+        console.error('Error fetching MongoDB album ID:', error);
+      });
+  }, [spotifyUrl]);
+  //console.log("Children:", children.type);
   return (
-    <CardContent style={{ marginTop: '1rem', width: '100%', display: 'flex' }}>
-      <iframe
-        title="Spotify Web Player"
-        src={spotifyEmbedUrl}
-        width={width}
-        height={height}
-        frameBorder={frameBorder}
-        style={{ borderRadius: 8, ...style, border: 'none' }}
-        allow={allow}
-        {...props}
-      />
-      {children && (
-        <div style={{ width: '100%', display: 'flex' }}>{children}</div>
-      )}
-    </CardContent>
+    <>
+      <CardContent
+        style={{
+          marginTop: '1rem',
+          width: '100%',
+          display: 'flex',
+          flexDirection: contentLayout,
+          justifyContent: 'center',
+          alignItems: 'center',
```

Comment:
Do we need everything centered horizontally and vertically; or just the children div?